keyboards/canary/canary60rgb/v2/config.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2022 DZTECH <moyi4681@live.cn>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define I2C1_SDA_PIN GP20
+#define I2C1_SCL_PIN GP21
+#define I2C_DRIVER I2CD0
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+#define IS31FL3733_I2C_ADDRESS_1 IS31FL3733_I2C_ADDRESS_GND_GND
+
+#ifdef RGB_MATRIX_ENABLE
+#    define USB_SUSPEND_WAKEUP_DELAY 5000
+#endif
```

Comment:
I checked with tofu\ii\v1 and tofu\jr\v1. These keyboards use the RP2040 with the LED driver is31fl3737, my keyboard use is31fl3733 which communicate to RP2040 using I2C too. Therefore, I think it is better to follow these 2 keyboards config because they are currently sold on the market.

Reply:
I removed the ifdef. Please check again.